template-parts/flexible-cpts/details.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Template part for displaying fleixble details. Displays custom fields and taxonomeis as list or tags.
+ */
+
+ $post_type = get_post_type();
+ 
+?>
+
+<div class="flexible-post-type-details">
+    <?php 
+    
+    $groups = acf_get_field_groups(array('post_type' => $post_type)); 
+    
+    if(is_array($groups) && count($groups) > 0){
+
+        foreach($groups as $group) {
+
+            $fields = acf_get_fields($group['key']);
+
+            if(is_array($fields) && count($fields) > 0){
+
+                foreach($fields as $field){
+                    
+                    $field_value = get_field($field['name']);
+
+                    if(!empty($field_value) && $field["single_view"]){
```

Comment:
I get an error message on this page "Undefined array key "single_view"".  When I look at $field["single_view"]  a get NULL. I can wrap a !empty() around $field["single_view"] to quiet it but that's kind of hidding the fact it is coming back null.